_posts/2020-06-21-covid-decision.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Will The Conference Go Ahead In 2020?
+description: "The Decision Around COVID-19"
+date: 2020-06-19
+cover: "/assets/header_image.jpg"
+author: Jessica White
+categories: [dddem]
+comments: false
+share: true
+---
+
+----
+<center>
+<h3 class="quote"><i>Our Final Decision And How We Reached It</i> </h3>
+</center>
+
+---
+<br/>
+
+The DDD East Midlands team have decided that the most responsible thing to do is **postpone the DDD East Midlands Conference until 2021**. Though this has been a tough decision to make, the safety of our delegates is of the upmost importance.
+
+<br/>
+
+# Why we came to this decision
+
+The team met every 4 weeks to assess the statistics around COVID-19 and what it would take to run a safe and successful conference. As two of our team are currently shielding, this also included the ability or organise the conference effectively.
+
+It looks likely that some form of social distancing will still be in place by the time October comes around. Not only is the overhead of ensuring social distancing is adhered to a logistical challenge, but we think it will have a big effect on the feel of the conference itself. 
+
+DDD East Midlands is a chance for the community and local businesses to celebrate and showcase all that makes the local tech community great. There are a range of activities planned for the conference and a lot of this includes some range of closeness or contact with surfaces. 
+
+The health and safety of our delegates, and that they have a fantastic experience of the event is really important to us.
+
+### The other options we considered.
```

Comment:
Consider re-wording to something like "Why we're not moving online"? as this is the only other option discussed